core/domain/exp_services.py

```diff
@@ -892,6 +892,11 @@ def _save_exploration(committer_id, exploration, commit_message, change_list):
                 exploration, state_names_with_unchanged_answer_groups,
                 exp_versions_diff)
 
+    # Trigger exploration issues model updation.
+    if feconf.ENABLE_PLAYTHROUGHS:
```

Comment:
Quick check: at which point exactly are we planning to flip this bit?

Reply:
@seanlip After the one-off job is run, we remove the checks in the backend so that for every new exploration and every new exploration commit, the empty model gets updated. The front end flag will be lifted at the end.

Reply:
Ah -- more specifically, is this job meant to be run during the June release? If so, please prepare a set of release instructions for @tjiang11, especially if this entails taking the server offline for a bit. (I assume that you don't want to have the server running with these flags not flipped, since then you'll have to re-run the one-off job later.)